### PR TITLE
P4 - Replace log from ethereum to bee's own log library

### PR DESCRIPTION
**Pull request by jmozah**
This PR replaces shed log library with bee logging library, to remove a dependency on swarm codebase.
Refer P3 in #66

### CODE REVIEW

File: pkg/shed/db.go [hunk 1]
```diff
@@ -81,6 +84,7 @@ func NewDB(path string, metricsPrefix string) (db *DB, err error) {
 func (db *DB) Put(key []byte, value []byte) (err error) {
 	err = db.ldb.Put(key, value, nil)
 	if err != nil {
+		db.logger.Debugf("failed to insert in to DB", "err", err)
```
Comment (janos) [inline pkg/shed/db.go:87]: This usage of Debugf is not correct.

Comment (janos) [reply pkg/shed/db.go:87]: @jmozah could you just fix this, and we can merge?

File: pkg/shed/db.go [hunk 2]
```diff
@@ -81,6 +84,7 @@ func NewDB(path string, metricsPrefix string) (db *DB, err error) {
 func (db *DB) Put(key []byte, value []byte) (err error) {
 	err = db.ldb.Put(key, value, nil)
 	if err != nil {
+		db.logger.Debugf("failed to insert in to DB. Error : ",  err.Error())
```
Comment (janos) [inline pkg/shed/db.go:87]: This is still not functionally right. Missing `%s`.

File: pkg/shed/db.go (hunk 1 above)
Comment (jmozah) [reply pkg/shed/db.go]: @janos oops... forgot this one...Dne it now...

File: pkg/shed/db.go (hunk 2 above)
Comment (jmozah) [reply pkg/shed/db.go]: oops.. sorry..

**Comment by jmozah** [comment]
@janos  Please merge this if you are satisfied.

**Pull request merged**